distribution/docs/src/main/resources/content/_managing/_configuring/global-settings.adoc

```diff
@@ -439,31 +439,20 @@ Thumbs.db
 |solr.client
 |String
 |Type of Solr configuration
-|HttpSolrClient
+|CloudSolrClient
 |Yes
 
-6+^h|Solr Cloud Properties
+6+^h|SolrCloud Properties
 
```

Comment:
Looks like you can also get rid of the `solr.mem` from `custom.system.properties` and from the documentation.